core/src/elo/elo.service.ts

```diff
@@ -261,4 +262,57 @@ export class EloService {
         const seriesTypeStr = series.matchParent.fixture ? "fixture" : series.matchParent.scrimMeta ? "scrim" : "unknown";
         return `\`seriesId=${seriesId}\` ${seriesTypeStr ? `(${seriesTypeStr})` : ""} successfully marked \`fullNcp=${isNcp}\` with updated elo, and all connected replays had their elo updated.${numReplays && dummiesNeeded ? ` **${dummiesNeeded} dummy replay(s)** were added to the series.` : ""}`;
     }
+
+    async prepMigrationData(): Promise<NewPlayer[]> {
+        // Run queries, first refreshing the materialized view
+        this.logger.verbose("Refreshing materialized view.");
+        // await this.dataSource.manager.query("REFRESH MATERIALIZED VIEW mledb.v_current_elo_values");
+
+        // Then querying from it
+        /* eslint-disable @typescript-eslint/no-unsafe-assignment,
+        @typescript-eslint/no-explicit-any */
+        this.logger.verbose("Querying the materialized view.");
+        const rawData: any[] = await this.dataSource.manager.query("SELECT player_id, elo, league, salary, name FROM mledb.v_current_elo_values");
```

Comment:
You should write an interface for what this `rawData` looks like to avoid using `any`

Reply:
Fixed